doc/modules/changes/20170510_anne-glerum

```diff
@@ -0,0 +1,6 @@
+Changed: The chunk geometry now removes the manifold id only from the boundaries and sets the correct
+boundary objects for dealii versions prior to version 9. For dealii 9, only the manifold is used,
```

Comment:
It is not in 8.5 yet, so we include compiler statements to test against deal.II 9 or newer (#if !DEAL_II_VERSION_GTE(9,0,0)).